pkg/system/filesys_windows.go

```diff
@@ -0,0 +1,64 @@
+// +build windows
+
+package system
+
+import (
+	"os"
+	"regexp"
+	"syscall"
+)
+
+// MkdirAll implementation that is volume path aware for Windows.
+func MkdirAll(path string, perm os.FileMode) error {
+	if re := regexp.MustCompile(`^\\\\\?\\Volume{[a-z0-9-]+}$`); re.MatchString(path) {
+		return nil
+	}
+
+	// The rest of this method is copied from os.MkdirAll and should be kept
+	// as-is to ensure compatibility.
```

Comment:
@swernli i'm not sure that answers my concern (or else i'm missed something).

What's the expected behavior with `MkdirAll("\?\Volume{3eb4b542-ef91-11e4-aa47-10604b6d8e85}\Foo")` ? To just return nil ? If that's the case, then why not do:
```
func MkdirAll(path string, perm os.FileMode) error {
	if re := regexp.MustCompile(`^\\\\\?\\Volume{[a-z0-9-]+}$`); re.MatchString(path) {
		return nil
	}
	return os.MkdirAll(path, perm)
}
```